src/components/budget/BudgetApprovalDashboard.tsx

```
@@ -0,0 +1,182 @@
+                                        onClick={() => handleApprove(req.id)}
+                                        disabled={actionLoading === req.id}
+                                        className="gap-1.5 bg-emerald-600 hover:bg-emerald-700"
+                                    >
+                                        {actionLoading === req.id ? (
+                                            <Loader2 className="h-4 w-4 animate-spin" />
+                                        ) : (
+                                            <Check className="h-4 w-4" />
+                                        )}
+                                        Approve
+                                    </Button>
+                                    <Button
```

ESLint

[error] Replace `····································` with `··················`

(prettier/prettier)